contrib/mkimage-arch.sh

```diff
@@ -14,6 +14,8 @@ hash expect &>/dev/null || {
 	exit 1
 }
 
+export LANG="en_US.UTF-8"
```

Comment:
What if someone doesn't have this particular locale on their local system?  Wouldn't `C.UTF-8` be more universal?